### PR TITLE
docs: Add section about working with github

### PR DESCRIPTION
**Pull request by tzhelyazkova**
This adds explanation about the github workflows we follow.
This is meant for anyone (dev, ux, pm) who contributes to the repository.

### CODE REVIEW

**Comment by raja-wmde** [comment]
That's a really nice explanation. Easy to understand. :)

**Review by raja-wmde — APPROVED** [review]
Great explanation.

**Review by itamargiv — APPROVED** [review]
Great intro, super clear.

File: docs/src/doc/01_getting_started.stories.mdx [hunk 1]
```diff
@@ -12,4 +12,40 @@ The documentation is organized into these categories:
...
+
+At least one person (different than the author of the PR) has to approve the PR in order for the merge button to become active.
+This is true regardless of the type of change - tokens, docs or vue-components.
+
+
+### Merging to master
+
+The master branch is the single source of truth for all three parts of the Design System - tokens, docs and vue-components.  
+Nevertheless, when you are developing a new feature, you can safely merge to master, because the Design System does not get released automatically on push to master.
+A release happens after several intentional actions a developer takes.
```
Comment (SaiSan-WMDE) [inline docs/src/doc/01_getting_started.stories.mdx:50]:
```suggestion
A release happens after a developer takes several intentional actions.  
```
Is this what the sentence means? No tea no shade

Comment (tzhelyazkova) [reply docs/src/doc/01_getting_started.stories.mdx:50]: Indeed, the original sentence doesn't make much sense 😆

**Review by SaiSan-WMDE — APPROVED** [review]
The content is super clear! I just added a nitpicky possible correction.

**Pull request merged**